modules/infinite-scroll/themes/twentyseventeen.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Infinite Scroll Theme Assets
+ *
+ * Register support for Twenty Seventeen.
+ */
+
+/**
+ * Add theme support for infinite scroll
+ */
+function twentyseventeen_infinite_scroll_init() {
```

Comment:
Hey @dereksmart. I have namespaced all functions for twentyseventeen.
I didn't add the prefix in the beginning, because the functions in other theme files don't have such( twentysixteen.php, twentyfifteen.php etc ) . So I think that it will be nice to add such prefix to all themes too. I can prepare a separate PR if you agree with me.